File: fg_metrics/VSTS/VSTSComms/VSTSComms/AgileSprint.cs
using System;
using System.Collections.Generic;

namespace VSTSComms
{
    public partial class AgileSprint
    {
        public string AgileSprintId { get; set; }
        public string AgileSprintName { get; set; }
        public string AgileSystemId { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string SprintDescription { get; set; }
    }
}

File: portal_backend/src/entities/agile_sprint.ts
import { Entity, PrimaryColumn, Column, ManyToOne, JoinColumn } from 'typeorm'
import { AgileSystem } from '../entities/agile_system'
@Entity({ name: 'agile_sprint' })
export class AgileSprint {
  @PrimaryColumn('uuid', { name: 'agile_sprint_id' })
  agileSprintId: string

  @Column('varchar', { name: 'agile_sprint_name' })
  agileSprintName: string

  @ManyToOne(type => AgileSystem)
  @JoinColumn({ name: 'agile_system_id' })
  agileSystem: AgileSystem

  @Column('varchar', { name: 'sprint_description' })
  sprintDscription: number

  @Column('varchar', { name: 'sprint_start_date' })
  sprintStartDate: Date

  @Column('varchar', { name: 'sprint_end_date' })
  sprintEndDate: Date
}
